tests/conftest.py

```diff
@@ -58,8 +58,10 @@ def create_driver(prep_properties, request):
 @pytest.hookimpl(tryfirst=True, hookwrapper=True)
 def pytest_runtest_makereport():
     outcome = yield
+    driver = dg.DRIVER
+    assert driver is not None, "Expected instance of a Web Driver but got None instead"
```

Comment:
why we need this?

Reply:
to pass it to the hook method for screen shots.
I still don't have a solution for driver instance passing to that method